File: aarTest/src/main/java/com/example/aartest/TestActivity.java
package com.example.aartest;

import android.app.Activity;
import android.os.Bundle;
import android.util.Log;


public class TestActivity extends Activity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.test_layout);
        findViewById(R.id.kill_btn).setOnClickListener(v -> {
            TestService ts = new TestService();
            String test = ts.getTestStr();
            String test2 = ts.testCall();
            test.toString();
            Log.d("jhlee", "test2: " + test2);
        });
    }
}

File: contentprovidertest/src/main/java/com/nhnace/contentprovidertest/TestActivity.kt
package com.nhnace.contentprovidertest

import android.os.Bundle
import android.util.Log
import androidx.appcompat.app.AppCompatActivity

class TestActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        Log.d("jhlee", "TestActivity Oncreate")
        finish()
    }
}